requirements.yml

```diff
@@ -0,0 +1,13 @@
+channels:
+  - conda-forge
+dependencies:
+  - python
+  - clang=16
+  - clang-format
+  - compiler-rt_linux-64
```

Comment:
why do we need compiler-rt?

Reply:
Mainly because it contains the sanitizer libraries.

Reply:
Btw, it seems impossible to find out a package given a file name (that supposedly belong to it).